### PR TITLE
new 2 PR

### PR DESCRIPTION
**Pull request by diagramphamola**

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -1 +1,9 @@
 aaaaaa
+
+
+ccccccccccc
```
Comment (diagramphamola) [inline test.txt]: rev

File: test.txt [hunk 2]
```diff
@@ -1 +1,9 @@
...
+
+
+vvvvvvvvvvvvvvvvvvvvvv
```
Comment (diagramphamola) [inline test.txt]: dfdsfds

File: test.txt [hunk 3]
```diff
@@ -1 +1,9 @@
 aaaaaa
+
+
+wwwwwccccccccccc
+
+
+vvvvvvvvvvvvvvvvvvvvvv
```
Comment (diagramphamola) [inline test.txt:7]: dddd

File: test.txt [hunk 4]
```diff
@@ -1 +1,9 @@
 aaaaaa
+
+
+wwwwwccccccccccc
```
Comment (diagramphamola) [inline test.txt:4]: fdgfdgfdg

File: test.txt [hunk 5]
```diff
@@ -1 +1,9 @@
 aaaaaa
+
+
```
Comment (diagramphamola) [inline test.txt:3]: sssssssssssssssssssssssss

File: test.txt [hunk 6]
```diff
@@ -1 +1,9 @@
 aaaaaa
+
```
Comment (freekman) [inline test.txt:2]: ddddd

Comment (freekman) [inline test.txt:2]: ddddddddd

**Pull request closed without merging**